plugins/tracker-resources/src/components/issues/IssuePreview.svelte

```diff
@@ -138,7 +138,6 @@
 <style lang="scss">
   .descr {
     overflow: hidden;
-    max-height: 25rem;
```

Comment:
May be limit to height of the screen?

Reply:
There is already a limitation in max-height for scrollerContent several lines below